scripts/check-lint

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I don't know how to verify this in the GitHub review UI, but make sure that this file is marked as Executable in git.

Reply:
Checking it with `git ls-files` gave following output: 
```
100755 bc378bf5f5ab7d384ab81f3ded1969e1802f8527 0       scripts/check-cla.sh
100755 513837e7aacd9cb7ce78a023ddeaa6fb87c5f4a5 0       scripts/check-lint.sh
```
Mode for both (renamed) files is includes executable flag